src/components/containers/recentArticles/recentArticles.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css?family=Playfair+Display');
+@import url('https://fonts.googleapis.com/css?family=Noto+Serif');
+@import url('https://fonts.googleapis.com/css?family=Sunflower:300');
+
+body {
+    padding: 0;
+    margin: 0;
+    /* font-family: 'Playfair Display', sans-serif; */
```

Comment:
shouldn't the font be the same?

Reply:
The fonts are the same. the font is coming from APP.scss.